docs/content/en/docs/community/contributing.md

```diff
@@ -5,80 +5,129 @@ description: >
+  * p2: not currently being tracked (backlog)
+* "status/\<status\>" - status of the issue (notstarted, implementation, etc.)
+* "kind/\<kind\>" - type of issue (bug, feature, enhancement, etc.)
+
+Refer to the [template](https://github.com/aws/eks-anywhere/tree/main/.github/ISSUE_TEMPLATE) for more information on
+what goes into an issue description.
+
+### GitHub Milestones
+
+GitHub Milestones are used to plan work that is currently being tracked.
+
+* next: changes for next release
+* next+1: won't make next release but the following
+* techdebt: used to keep track of techdebt items, separate ongoing effort from release action items
+* oncall: used to keep track of issues needing active follow-up
+* backlog: items that don't have a home in the others
+
+### GitHub Projects (or tasks within a GitHub Issue)
+
+GitHub Projects are used to keep track of bigger features that are made up of a collection of issues.
+Certain features can also have a tracking issue that contains a checklist of tasks that
+link to other issues.
 
 ## Reporting Bugs/Feature Requests
 
-We welcome you to use the GitHub issue tracker to report bugs or suggest
-features.
+We welcome you to use the GitHub issue tracker to report bugs or suggest features that have actionable items/issues
+(as opposed to introducing a feature request on GitHub Discussions).
 
-When filing an issue, please check
-[existing open](https://github.com/aws/eks-anywhere/issues) or
-[recently closed](https://github.com/aws/eks-anywhere/issues?q=is%3Aissue+is%3Aclosed),
-issues to make sure somebody else hasn't already reported the issue. Please
-try to include as much information as you can. Details like these are
-incredibly useful:
+When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
+reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
 
 * A reproducible test case or series of steps
-* The version of the code being used
+* The version of our code being used
 * Any modifications you've made relevant to the bug
 * Anything unusual about your environment or deployment
 
 ## Contributing via Pull Requests
-Contributions via pull requests are much appreciated. Before sending us a
-pull request, please ensure that:
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
 
 1. You are working against the latest source on the *main* branch.
-2. You check existing open, and recently merged, pull requests to make sure
-someone else hasn't addressed the problem already.
-3. You open an issue to discuss any significant work - we would hate for your
-time to be wasted.
+2. You check existing open, and recently merged, pull requests to make sure someone else hasn't addressed the problem already.
+3. You open an issue to discuss any significant work - we would hate for your time to be wasted.
```

Comment:
We should switch all of the numbers to markdown style with the same number so items can more easily be reordered if necessary
```
1.
1.
1.
```
will render as 
1.
2.
3.

Reply:
But would it render as such on GitHub markdown?

Reply:
Cool confirmed